package.json

```diff
@@ -22,15 +22,12 @@
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.cjs"
+      "import": "./dist/index.mjs"
     },
     "./nuxt": {
-      "import": "./dist/nuxt/module.mjs",
-      "require": "./dist/nuxt/module.cjs"
+      "import": "./dist/nuxt/module.mjs"
     }
   },
-  "main": "./dist/index.cjs",
   "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
   "typesVersions": {
```

Comment:
since we're not using `.js` in the file extensions in the package, we need to add the `d.mts` for nuxt , `"./*" isn't effective:
```json
"typesVersions": {
  "*": {
    "nuxt": [
      "./dist/nuxt/module.d.mts"
    ]
  }
}
```

Reply:
So should we have the following?

```json
  "typesVersions": {
    "*": {
      "*": [
        "./dist/*",
        "./*"
      ],
      "nuxt": [
        "./dist/nuxt/module.d.mts"
      ]
    }
  },
```

Reply:
or just use `.js` instead `.mjs` and keep current "typesVersions"

Reply:
> So should we have the following?
> 
> ```json
>   "typesVersions": {
>     "*": {
>       "*": [
>         "./dist/*",
>         "./*"
>       ],
>       "nuxt": [
>         "./dist/nuxt/module.d.mts"
>       ]
>     }
>   },
> ```

no, just nuxt entry

Reply:
in fact Nuxt will drop support for node10 module resolution (if not yet removed), we can remove "typesVersions" entry

Reply:
Nuxt still supports node 10 , so it is fine